### ISSUE TITLE
13062022 Nightly build v1.77.3

### ISSUE THREAD

**Issue by Kvarkas** [mRemoteNG/mRemoteNG#2245]
Welcome to comment there if you want to say something or blame me for slow work! 
![book](http://www.kolobok.us/smiles/artists/vishenka/l_book.gif)

**Comment by Gp0n5** [comment]
Not saving Options

**Comment by Kvarkas** [comment]
yes its due backup option changing where settings file will be saved, in couple of days hope to finish such to fix.

**Comment by dimangelid** [comment]
Hello @Kvarkas , will you create an installable version of the new nightly build? It will be very useful since it is more comfortable for many of us.

**Comment by Kvarkas** [comment]
I may, but it was opposite request actually :D because it may change a quite often (I know now its not happening every night :P) so from portable you can just copy .exe file as nothing else currently should change so its may enoght to update. But its not a problem to include installation files as well (will do on next iteration)

**Comment by dimangelid** [comment]
> I may, but it was opposite request actually :D because it may change a quite often (I know now its not happening every night :P) so from portable you can just copy .exe file as nothing else currently should change so its may enoght to update. But its not a problem to include installation files as well (will do on next iteration)

Thank you @Kvarkas !

Please also note that the portable version, creates a folder for the program settings under the user profile folder (for example C:\Users\User\AppData\Local and C:\Users\User\AppData\Roaming) . With this approach, the portable version can not be stored in a USB stick and be used at different computers.

Finally, i also have the issue mentioned by other users that the settings are not being stored.

**Comment by Kvarkas** [comment]
Currently we don't have a flow to produce true portable app (in meaning of portable apps terms) but that was several request already to make it such so once reorganise more important stuff will definitely add such ability

**Comment by Shekelme** [comment]
The program `mRemoteNG.exe` does not start. Windows 7 Ultimate x64 [Version 6.1.7601]

**Comment by simonai1254** [comment]
> The program `mRemoteNG.exe` does not start. Windows 7 Ultimate x64 [Version 6.1.7601]

Windows 7 is no longer supported by mRemoteNG, please update your system to a newer Windows version.

**Comment by Shekelme** [comment]
I hope that never happens.)

**Comment by Kvarkas** [comment]
well we want to move on and use modern technologies not stay in stone age with stone axes and legacy windows 95 ish

**Comment by dimangelid** [comment]
@Kvarkas 

Do you have any plans for releasing a new version that fixes the issue that does not save the settings?

**Comment by Kvarkas** [comment]
of course! but that will be premium option! :D

**Comment by journeym** [comment]
Any stable version coming in foreseeable future?

**Comment by Cool34000** [comment]
Keeps asking for .NET 6 even after installing it and after a reboot

![image](https://user-images.githubusercontent.com/17827397/175051157-3e4af7f3-ca4b-4fc5-81c2-5a5a1b1cd44c.png)

**Comment by simonai1254** [comment]
> Keeps asking for .NET 6 even after installing it and after a reboot
> 
> ![image](https://user-images.githubusercontent.com/17827397/175051157-3e4af7f3-ca4b-4fc5-81c2-5a5a1b1cd44c.png)

You need the Desktop Runtime, the (core) runtime unfortunately is not enough

**Comment by Cool34000** [comment]
Thanks, I didn't see that there were 2 versions (core & desktop)

**Comment by atlib77** [comment]
Would really like to see this merged to the nb:
https://github.com/mRemoteNG/mRemoteNG/issues/1546

**Comment by scubamount** [comment]
Hello, options still not saving. Unusable until I can save options, thank you!

**Comment by Kvarkas** [comment]
I was mentioned that before, nb is not for work so disabled saving specially :)

**Comment by Kvarkas** [comment]
just to update almost complete conversion from windows forsms to wpf so soon expect a new nb ;)

**Comment by TomLiu-GitHub** [comment]
I give up 'Nightly Build 'because there are too many errors

**Comment by uSlackr** [comment]
> just to update almost complete conversion from windows forms to wpf so soon expect a new nb ;)

Any time soon?  The lackof saving options makes this unusable

**Comment by kbjc11686** [comment]
It sounds like a complete conversion from WinForms to WPF is more of a major upgrade (v2.0... or 1.8 or some such). It seems to me that the most prudent path, time permitting, is to write a final stable version with installer and with the save options bug corrected first.

**Comment by Kvarkas** [comment]
not correct, most work is happening to organise settings as they are complicated, messy and partially not acting as expected. WPF implementation is only small bit of such and soon will be next nb release with saving enabled

**Comment by kbjc11686** [comment]
Understood, and that makes sense. Sounds like good progress!

**Comment by W1sk3r** [comment]
I'm excited for this next build. Look forward to testing.

**Comment by andrei-micuda** [comment]
Love the progress. I'm eyeing a new stable release for the Dynamic Resolution Support.

**Comment by Kvarkas** [comment]
its depends :)

**Comment by Xhanti** [comment]
Hi, is there any place to see progress towards the next stable release or is this the place to check on updates here?
https://github.com/mRemoteNG/mRemoteNG/projects/3

**Comment by LIyoai** [comment]
I would expect it under milestones
https://github.com/mRemoteNG/mRemoteNG/milestones

**Comment by Xhanti** [comment]
Thanks @LIyoai

**Comment by Kvarkas** [comment]
yes, may need update milestones for better visability of what going on, will try once have a bit more time, at the moment puttyng updated to latest 7.8 release and bussy with testing to release a new nb

**Comment by Xhanti** [comment]
Are the help wanted items up to date? I would like to help chip away at issues that are blocking the next release if possible

**Comment by Kvarkas** [comment]
if you could review issues taged 1.77.3 (2) and may have a look if something you can code better...

**Comment by wewenttothemoon** [comment]
@Kvarkas Thank you for your work on this. I just wanted to see if there was any update regarding the previous posts about #2288 

Is there any work around we can perform? (I know you mentioned you disabled this specifically, but I am just curious)

**Comment by Kvarkas** [comment]
its related to complete rework how settings are saved so its tought to passto somebody else, need to complete to desired point then we may have improve it better. and save is disabled to avoid even bigger mess ;)

**Comment by rockmv** [comment]
Hello, i have a problem with the conf-file. The 1.77.3NB Portable save no entries in the file.

**Comment by simonai1254** [comment]
Hi 

1.77.3-nb does not save config files, it is disabled in purpose for development reasons. (See #2245)

Regards

**Comment by FingerlessGlov3s** [comment]
Once it saves Config files, I can test and look for bugs 😊

**Comment by q16marvin** [comment]
hey im little confused, i dont get it :) for what reason 1.77.3 NB is not saving config?

**Comment by Kvarkas** [comment]
its a big chunk of recode of a way how we save settings is on going, should be released soon

**Comment by sovamind** [comment]
Is there a nightly version that DOES save settings?  

I need the Chromium rendering engine support which is in the nightly builds but is not in the stable release.  

I'm also curious why haven't any of the things in nightly that are working been moved to the Stable release?  The Beta release doesn't exist anymore (or download links are broken on website) and the Stable release hasn't been updated since 2019.  This leaves us only with the current nightly builds which now have this issue of not saving settings.

**Comment by sovamind** [comment]
I also agree with others that have stated that a "portable" installation should not be saving settings in the %USERPROFILE% directory but in the same folder as the application so that it would be compatible with running on USB drives and other portable app systems.  

Or maybe the application settings should just be stored in the connections file?  This would be helpful for those of us that work with several clients systems as then the default username option (and some other options) in the application settings can be stored in the connections file and we can just run multiple instances of mRemoteNG for each client's systems.  This would actually be a huge improvement.

**Comment by nserup** [comment]
This program is simply the best, but even better - You are continuously updating it, no even better you are making it free so i can use it on my work without discussing it with my boss, no no no even better, you are making a dark mode, which my eyes are loving and will thank you for! 
I will definitely be making a donation to show appreciation for the great project 
I have already recommended it for my sysadm colleages, which agree this is the best program to use for RDP, SSH, HTTPS🙏💪👍👌

**Comment by Kvarkas** [comment]
thanks for warm words! :) And if you sysadmin collages have wishes what they need to simplify they work - don't hesitate to tell me :)

**Comment by Helix751** [comment]
`winget` is listing 1.77.3 as an upgradeable version from the Nightly channel, even though it lacks saving options.

Given it is non-production and lacks significant features from former (1.77.2), it is not an upgrade and should not be listed in this upgrade list.

For the time being I had to force excluding mRemoteNG from `winget` (`winget pin add mRemoteNG.mRemoteNG.Nightly`) to avoid accidentally "upgrading" and losing functionality.

**Comment by journeym** [comment]
Wow, a build from 2022, nice

**Comment by Kvarkas** [comment]
> `winget` is listing 1.77.3 as an upgradeable version from the Nightly channel, even though it lacks saving options.
> 
> Given it is non-production and lacks significant features from former (1.77.2), it is not an upgrade and should not be listed in this upgrade list.
> 
> For the time being I had to force excluding mRemoteNG from `winget` (`winget pin add mRemoteNG.mRemoteNG.Nightly`) to avoid accidentally "upgrading" and losing functionality.

Hello, what not me who add this to rotation :(

**Comment by Kvarkas** [comment]
> Wow, a build from 2022, nice

we have builds from 2025 as well, but what only for premium holders :D

**Comment by Kvarkas** [comment]
not relevant
